packages/cli/src/run.ts

```diff
@@ -43,7 +40,7 @@ export async function runCmd(args: string[], cwd: string) {
     throw new ExitError(1);
   } else {
     const { code } = await spawn("yarn", args.slice(1), {
-      cwd: matchingPackages[0].dir,
+      cwd: path.join(rootDir, matchingPackages[0].relativeDir),
```

Comment:
most likely this can be simplified here too
```suggestion
      cwd: matchingPackages[0].dir,
```